packages/toolkit/src/query/createApi.ts

```diff
@@ -242,6 +252,7 @@ export function buildCreateApi<Modules extends [Module<any>, ...Module<any>[]]>(
       refetchOnMountOrArgChange: false,
       refetchOnFocus: false,
       refetchOnReconnect: false,
+      invalidationBehavior: 'delayed',
```

Comment:
based on conversations regarding this MR, this new default behaviour is the main sticking point for this PR.

if the default was switched back to the existing behaviour (though obviously keeping the ability to enable the new behaviour), this would very easily fit into another 1.9 release - we could then switch to the new behaviour as a default in 2.0, where changes in behaviour like this could be more expected.

alternatively, this entire change could be held off until 2.0, but considering we don't expect to be releasing 2.0 stable for a good while (realistically, multiple months), that seems like a disservice to the people interested in this change.

Reply:
```suggestion
      invalidationBehavior: 'immediately',
```

Okay, so we should just apply the above suggestion and then just merge already?

Reply:
yeah - CI would hopefully check that there aren't any tests etc that would fail with the default being changed back